tests/CMakeLists.txt

```diff
@@ -370,10 +370,30 @@ endif()
 string(REPLACE "test_" "${CMAKE_CURRENT_SOURCE_DIR}/test_" PYBIND11_ABS_PYTEST_FILES
                "${PYBIND11_PYTEST_FILES}")
 
+if(NOT PYBIND11_MEMCHECK)
```

Comment:
This can only be set at configure time? Is there no way to pass information at build time?